### PR TITLE
CSHARP-3074: Update year in documentation

### PR DESCRIPTION
**Pull request by vincentkam**
https://jira.mongodb.org/browse/CSHARP-3074

### CODE REVIEW

File: Docs/landing/layouts/partials/footer.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="footer">
   <div class="copyright">
-    <p>Copyright © 2015 <a class="smalltext" href="http://www.mongodb.com">MongoDB, Inc</a>.
+    <p>Copyright © 2020 <a class="smalltext" href="http://www.mongodb.com">MongoDB, Inc</a>.
```
Comment (DmitryLukyanov) [inline Docs/landing/layouts/partials/footer.html]: maybe use JS to calculate this date and avoid this task each year?

Comment (DmitryLukyanov) [reply Docs/landing/layouts/partials/footer.html]: I see that we already use js https://github.com/vincentkam/mongo-csharp-driver/blob/fb39143e30a085eafc55154f67bcbb64051ff15b/Docs/landing/static/404.html#L81 but not sure that it's expected address :)

Comment (vincentkam) [reply Docs/landing/layouts/partials/footer.html]: Done

**Comment by vincentkam** [comment]
Ready for re-review!

**Review by DmitryLukyanov — APPROVED** [review]
LGTM

**Pull request merged**